src/main/resources/org/jenkinsci/plugins/authorizeproject/form/dropdownDescriptorSelector.jelly

```diff
@@ -39,7 +39,7 @@ THE SOFTWARE.
     <st:attribute name="title" use="required">
       Human readable title of this control.
     </st:attribute>
-    <st:attribute name="lazy">
+    <st:attribute name="capture">
```

Comment:
Make compatible with `dropdownDescriptorSelector` of the latest Jenkins core.
https://github.com/jenkinsci/jenkins/blob/jenkins-1.645/core/src/main/resources/lib/form/dropdownDescriptorSelector.jelly